docs/en/modules/admin/pages/features/notifications.adoc

```diff
@@ -19,109 +19,114 @@ image:features/notifications/no_notifications_yet.png[Example of notifications p
 
 == Notifications list
 
-Below is an exhaustive list of actions that trigger notifications to participants. Participants are divided into three categories: assigned users, followers and administrators.
+Below is an exhaustive list of actions that trigger notifications to participants. Participants are divided into three categories: affected users, followers, and administrators.
 
-[options="header"]
+[NOTE]
+====
+Affected user's depend in the notification context. For instance it could be the author of a moderated content, a user that earns a badge, a user that has been mentioned, etc.
```

Comment:
```suggestion
Affected users depend on the notification context. For instance, it could be the author of moderated content, a user that earns a badge, a user that has been mentioned, etc.
```